source/internal/_clone.js

```diff
@@ -83,7 +88,11 @@ _ObjectMap.prototype.set = function(key, value) {
 _ObjectMap.prototype.hash = function(key) {
   let hashedKey = [];
   for (var value in key) {
-    hashedKey.push(key[value]);
+    if (_hasPrototype(key[value])) {
+      hashedKey.push(key[value]);
+    } else {
+      hashedKey.push(Object.prototype.toString.call(key[value]));
+    }
```

Comment:
What is the use-case that requires this block?  I'm not following something.

Reply:
@CrossEye All the values are added in `hashedKey` and `hashedKey.join` tries to convert all the elements by calling `.toString` on it, if necessary. In our case, the object without prototype doesn't have `toString`. So, handling that case with `Object.prototype.toString.call`. I was not sure about the implications of doing `Object.prototype.toString.call` on all the elements of the `hashedKey`. That's why it is done this way.